pegasus/sites.v3/hourofcode.com/i18n/en.yml

```
@@ -2561,7 +2561,7 @@
   tutorial_vidcodemagic_longdescription: "Make flashes appear randomly around the screen using a loop."
   tutorial_codespeakmeal_longdescription: "Learn to code an app that helps users make healthy food decisions."
   tutorial_itchfish_longdescription: "Scratch 3.0 tutorial for making a game with a shark chasing and eating fish disrupted when a diver is eaten accidentally."
-  tutorial_bsdwebsite_longdescription: "This course allows students to use code as a way to share what is important or interesting to them.They can follow the lesson or create unique projects while learning HTML, CSS and JavaScript.They are guided with video tutorials, live coding and access to the code glossary. 'The My Favourites Website focuses on creating a webpage of the top 3 things you are interested in or love doing. "
+  tutorial_bsdwebsite_longdescription: "This course allows students to use code as a way to share what is important or interesting to them.They can follow the lesson or create unique projects while learning HTML, CSS and JavaScript.They are guided with video tutorials, live coding and access to the code glossary. 'The My Favourites Website focuses on creating a webpage of the top 3 things you are interested in or love doing. #codeisyourvoice"
```

Adding missing spaces
```suggestion
  tutorial_bsdwebsite_longdescription: "This course allows students to use code as a way to share what is important or interesting to them. They can follow the lesson or create unique projects while learning HTML, CSS and JavaScript. They are guided with video tutorials, live coding and access to the code glossary. 'The My Favourites Website focuses on creating a webpage of the top 3 things you are interested in or love doing. #codeisyourvoice"
```